MekHQ/src/mekhq/campaign/rating/CampaignOpsReputation.java

```diff
@@ -494,110 +497,32 @@ public int getTransportValue() {
-            doubleExcessCapacity = false;
-        } else if (getHeavyVeeBayCount() < getHeavyVeeCount()) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getHeavyVeeBayCount() < getHeavyVeeCount() * 2) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
-        }
-        heavyVeeBays -= getHeavyVeeBayCount();
-        int lightVeeBays = getLightVeeBayCount() + heavyVeeBays;
-        if (lightVeeBays == getLightVeeCount()) {
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (lightVeeBays < getLightVeeCount()) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (lightVeeBays < getLightVeeCount() * 2) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
-        }
-        if (getFighterBayCount() == getFighterCount()) {
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getFighterBayCount() < getFighterCount()) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getFighterBayCount() < getFighterCount() * 2) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
-        }
-        if ((getBaBayCount()) == getBattleArmorCount() / 5) {
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if ((getBaBayCount()) < getBattleArmorCount() / 5) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if ((getBaBayCount() * 2) < 2 * getBattleArmorCount() / 5) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
-        }
-        if (getInfantryBayCount() < getInfantryCount() / 28) {
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getInfantryBayCount() < getInfantryCount() / 28) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getInfantryBayCount() < getInfantryCount() / 14) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
-        }
-        if (getSmallCraftBayCount() == getSmallCraftCount()) {
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getSmallCraftBayCount() < getSmallCraftCount()) {
-            fullCapacity = false;
-            excessCapacity = false;
-            doubleExcessCapacity = false;
-        } else if (getSmallCraftBayCount() < (getSmallCraftCount() * 2)) {
-            fullCapacity = false;
-            doubleExcessCapacity = false;
+        TransportCapacityIndicators tci = new TransportCapacityIndicators();
```

Comment:
Refactored this code (not sure it was really working properly prior anyway) to unify the "sufficient/excess/double" transport capacity computation. This also had the effects of fixing the unit tests.